src/main/java/com/qaprosoft/carina/demo/mobile/gui/mfp/pages/android/MoreMenuPage.java

```diff
@@ -0,0 +1,24 @@
+package com.qaprosoft.carina.demo.mobile.gui.mfp.pages.android;
+
+import com.qaprosoft.carina.core.foundation.utils.factory.DeviceType;
+import com.qaprosoft.carina.demo.mobile.gui.mfp.pages.common.CommonPageBase;
+import com.qaprosoft.carina.demo.mobile.gui.mfp.pages.common.MoreMenuPageBase;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+
+@DeviceType(pageType = DeviceType.Type.ANDROID_PHONE, parentClass = MoreMenuPageBase.class)
+public class MoreMenuPage extends MoreMenuPageBase {
+    public MoreMenuPage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Override
+    public void AssertAllOptionsArePresent() {
```

Comment:
we don't need this method, all validations usually are just in test. think about other way to check it with few row in code